api/sshRekey.go

```diff
@@ -68,7 +68,7 @@ func (h *caHandler) SSHRekey(w http.ResponseWriter, r *http.Request) {
 
 	newCert, err := h.Authority.RekeySSH(ctx, oldCert, publicKey, signOpts...)
 	if err != nil {
-		WriteError(w, errs.ForbiddenErr(err))
+		WriteError(w, errs.ForbiddenErr(err, "error signing ssh certificate"))
```

Comment:
should this be `error rekeying ssh certificate`?

Reply:
At the end of the day, rekey is really a sign, and I believe signing is clearer. In any case, this is the fallback message, in case we don't have a better one from an inner package.

Reply:
`renew` is also a sign. My main goal is for it to not be confusing where in the code errors are coming from. If a bunch of different catch all errors direct people to the `sign` methods I think that could be confusing.

Reply:
Fixed on 0cebde3